src/main/java/com/napier/sem/Record.java

```diff
@@ -123,15 +122,35 @@ public Record(String name, String country, String district, long population)
         recordType = App.CITY;
     }
 
+    //Population Residence Report constructor, just used for testing purposes.
```

Comment:
missing space after comment, can be ignored and fixed later if there are no major changes we'd like to make to this branch